core/src/focus_tracker.rs

```diff
@@ -100,6 +100,8 @@ impl<'gc> FocusTracker<'gc> {
             tab_order.first()
         };
 
-        self.set(next.copied(), context);
+        if next.is_some() {
+            self.set(next.copied(), context);
```

Comment:
To be sure, when can this happen? As in... only if previously the entire list was non-empty and we had a focused item, but now is entirely empty?

Reply:
This happens when there is a focus, but tab ordering is empty, i.e. someone tabbed and tab ordering changed (e.g. setting `tabEnabled = false`), or the focus was set programmatically (`Selection.setFocus`).

This is tested by `tab_ordering_children`, when you comment out this `if` condition it will fail.